pyk/pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
 name = "pyk"
-version = "0.1.779"
+version = "6.3.0"
```

Comment:
There's also `K_VERSION` in `pyk/src/pyk/__init__.py`, which for now we can simply replace by `__verison__`.